--- Corefile-dist	2018-12-01 02:11:16.000000000 +0000
+++ Corefile	2018-12-01 02:20:43.000000000 +0000
@@ -13,4 +13,12 @@
     cache 30
     loadbalance
     reload
+    rewrite stop {
+        name regex ([^.]+)\.([^.]+)\.hlf\.blockdaemon\.io\. {1}-{2}.hlf-service-network.svc.cluster.local.
+        answer name ([^-]+)\-([^-]+)\.hlf-service-network\.svc\.cluster\.local\. {1}.{2}.hlf.blockdaemon.io.
+    }
+    rewrite stop {
+        name regex (.*)\.hlf\.blockdaemon\.io\. {1}.hlf-service-network.svc.cluster.local.
+        answer name (.*)\.hlf-service-network\.svc\.cluster\.local\. {1}.hlf.blockdaemon.io.
+    }
 }
